pkg/op/op.go

```diff
@@ -166,6 +166,11 @@ func NewOpenIDProvider(ctx context.Context, config *Config, storage Storage, opO
 
 	o.crypto = NewAESCrypto(config.CryptoKey)
 
+	// Avoid potential race conditions by calling these early
+	_ = o.AccessTokenVerifier()
+	_ = o.JWTProfileVerifier()
+	_ = o.openIDKeySet()
```

Comment:
wouldn't this also have to be done with for the `IDTokenHintVerifier`as well?

Reply:
You're quite right.  Thanks!